pysmt/cmd/install.py

```diff
@@ -229,10 +229,14 @@ def main():
             else:
                 print("set PYTHONPATH=" + bindings_dir + ";%PYTHONPATH%")
                 print("set PATH=" + bindings_dir + ";%PATH%")
+        elif platform.system().lower() == "darwin":
+            print("export PYTHONPATH=\"" + bindings_dir + ":${PYTHONPATH}\"")
+            print("export PATH=\"" + bindings_dir + ":${PATH}\"")
```

Comment:
why is PATH needed here?

Reply:
For Z3, it seems that on OSX you need to have the dylib files in the PATH instead of LD_LIBRARY_PATH....